spec/acceptance/suites/gce/nodesets/default.yml

```diff
@@ -0,0 +1,21 @@
+HOSTS:
+  el7:
+    roles:
+      - server
+      - default
+      - master
+    platform:   rhel-7-v20170426
+    hypervisor: google
+  el6:
+    roles:
+      - client
+    platform:   rhel-6-v20170426
+    hypervisor: google
+CONFIG:
+  log_level: verbose
+  type:      aio
+  keyfile:  ADD_ME
```

Comment:
Beaker doesn't have a way to work around this *yet* and we don't want to lose this test.